packages/dos-donts-block/src/DoDontItem.tsx

```diff
@@ -20,322 +20,308 @@ import {
-                                    'tw-mr-2 tw-w-auto tw-flex tw-items-center',
-                                    !title ? 'tw-opacity-30' : '',
-                                ])}
-                            >
-                                <IconComponent
-                                    type={type}
-                                    hasCustomDoIcon={hasCustomDoIcon}
-                                    doIconChoice={doIconChoice}
-                                    doIconAsset={doIconAsset}
-                                    hasCustomDontIcon={hasCustomDontIcon}
-                                    dontIconChoice={dontIconChoice}
-                                    dontIconAsset={dontIconAsset}
-                                />
-                            </div>
-                        )}
-                        <div className="tw-w-full tw-flex tw-items-center">
-                            <span
-                                style={{
-                                    marginBottom: 0,
-                                    marginTop: 0,
-                                    width: '100%',
-                                    lineHeight: 1,
-                                    display: 'inline-flex',
-                                }}
-                            >
-                                <textarea
-                                    rows={1}
-                                    ref={titleRef}
-                                    onChange={(event) => onChangeLocalItem(id, event.target.value, 'title')}
-                                    onBlur={() => onChangeItem(id, title, 'title')}
-                                    style={
-                                        {
-                                            ...BlockStyles.heading3,
-                                            marginBottom: 0,
-                                            marginTop: 0,
-                                            color: headingColor,
-                                            '--placeholder-color': headingColor,
-                                            '-webkit-text-fill-color': headingColor,
-                                        } as CSSProperties
-                                    }
-                                    value={title}
-                                    disabled={!editing}
-                                    aria-label="Title"
-                                    placeholder={editing ? 'Add a title' : ''}
-                                    className="tw-text-s tw-pointer-ev tw-w-full tw-placeholder-[var(--placeholder-color)] placeholder:tw-opacity-30 tw-placeholder-opacity-30 tw-bg-transparent tw-resize-none tw-text-text-weak tw-break-words tw-outline-none tw-whitespace-pre-wrap"
-                                />
-                            </span>
+                    {style === DoDontStyle.Icons && (title || body || editing) && (
```

Comment:
body is always there I think because it's the RTE value

Reply:
Initially it's an empty string which is falsy. That is why when you add an empty item the icons don't show up in view mode.

Reply:
Logic seems inconsistent though as if you type into the RTE and then backspace it will always appear after that

Reply:
Yep, changed the condition to `hasRichtextValue(body)`